components/styledComponents/navigation/NavigationList.tsx

```diff
@@ -0,0 +1,54 @@
+import * as NavigationMenu from "@radix-ui/react-navigation-menu";
+import { styled } from "../../../styles/stitches.config";
+
+export const NavigationList = styled(NavigationMenu.List, {
+  display: "flex",
+  //alignItems: "center",
```

Comment:
Unused code. It's good practice to not keep uncommented code. (https://henrikwarne.com/2017/04/28/code-rot/)